bin/cljfmt.sh

```diff
@@ -1,7 +1,21 @@
 #! /usr/bin/env bash
 
-set -euxo pipefail
+# Reformat only staged files OR all files if --all is passed.
```

Comment:
Can you add some comments to these files about how to invoke these scripts? I'm unclear on how I would run this on just a single file

Reply:
I've renamed the files and made them explicit that they do one thing only -- run their specific job on only modified or added files.